controllers/comments.controllers.js

```diff
@@ -1,9 +1,26 @@
+const {
+  insertComment,
+  selectCommentsByArticleId,
+} = require("../models/comments.models");
 const { checkExists } = require("../db/seeds/utils");
-const { selectCommentsByArticleId } = require("../models/comments.models");
+
+exports.postComment = (req, res, next) => {
+  const newComment = req.body;
+  const { article_id } = req.params;
+
+  insertComment(newComment, article_id)
+    .then((comment) => {
+      res.status(201).send({ comment });
+    })
+    .catch(next);
+};
 
 exports.getCommentsByArticleId = (req, res, next) => {
   const { article_id } = req.params;
-  const articlePromises = [selectCommentsByArticleId(article_id), checkExists("articles", "article_id", article_id)];
+  const articlePromises = [
```

Comment:
I think because article_id is a foriegn key PSQL will give you a nice status code if you adding a comment with a bad id